packages/flutter_tools/lib/src/device.dart

```diff
@@ -167,6 +167,35 @@ class DeviceManager {
     }
     return diagnostics;
   }
+
+  /// Find and return the List of target [Device] based upon currently connected
```

Comment:
Nit: follow dartdoc convention with a single line sentence to start.

`Find and return a list of devices based on the current project and environment`

Or something like that

Reply:
Done.